README.md

```diff
@@ -1,6 +1,16 @@
 # flusion
+
 Influenza forecasting using data fusion
 
+## Repository organization
+
+This repository has the following directories:
+
+- `code/`: all code for flusion and its component models, exploratory data analyses, and so on. See the readme in that folder for further information.
+- `data-raw/`: raw data measuring influenza activity, pulled from various sources. See the readme in that folder for further information.
```

Comment:
noticed that the `data-raw/` directory doesn't have a readme itself, though its sub-folders of each source do. Is this intentional?